provisioner/roles/configure_qradar_instance/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
@maxamillion I thought those mentions of private IP address were vital to the interna of QRadar? Isn't that why we created a fake interface as bridge in the image in the first place?